src/LongTermStorage/DotnetEventProcessor/ColdStorage.EventProcessor.ConsoleHost/ColdStorage.EventProcessor.ConsoleHost.csproj

```diff
@@ -36,16 +36,16 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.Data.Edm, Version=5.6.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Data.Edm.5.6.4\lib\net40\Microsoft.Data.Edm.dll</HintPath>
+      <Private>True</Private>
```

Comment:
This is something that happened after executing Update-Package –reinstall (to re-target a project I initially created as net45 but should be net451